### ISSUE TITLE
Explore other CI options

### ISSUE THREAD

**Issue by Stratus3D** [asdf-vm/asdf#593]
We are currently using Travis, and also recently added AppVeyor. We should explore other options like https://cirrus-ci.org/ that also support Windows.

**Comment by vic** [comment]
Github actions have been working great for the repos on `asdf-community`.

One particular thing I liked about GH actions is that, there seems to be no rate-limit for API calls. Perhaps since it's all running on GH environment? And many plugins just install from github releases, and some time ago it was this rate-limit on travis which was causing much plugins to fail if people didn't correctly set a GITHUB_API_TOKEN.

I'd go for using GitHub Actions since development already happens here. And most likely it already [supports Windows](https://help.github.com/en/github/automating-your-workflow-with-github-actions/about-github-actions#virtual-environment)

**Comment by imbsky** [comment]
Here are some limits list of GitHub Actions. It has a significant advantage over other CI providers.

> - You can execute up to 20 workflows concurrently per repository.
> - You can execute up to 1000 API requests in an hour across all actions within a repository.
> - Each job in a workflow can run for up to 6 hours of execution time.

> One particular thing I liked about GH actions is that, there seems to be no rate-limit for API calls.

Probably no, we need an API key to call the API, and I failed the test several times with that. but we no longer need to create and manage our own keys. I actually set it up like this.

https://github.com/asdf-community/asdf-direnv/blob/master/.github/workflows/workflow.yml#L41

https://help.github.com/en/github/automating-your-workflow-with-github-actions/virtual-environments-for-github-actions#github_token-secret
